pyam/utils.py

```diff
@@ -129,6 +129,18 @@ def format_data(df):
         missing = list(set(IAMC_IDX) - set(df.columns))
         raise ValueError("missing required columns `{}`!".format(missing))
 
+    if 'notes' in df.columns:
+        logger().info('Ignoring notes column in dataframe')
+        df.drop(columns='notes',inplace=True)
+        # remove SSP Public Database row
+        database_label_rows = [
```

Comment:
I'd suggest to drop all models starting with `@`, because there are many other (non-public) database instances from which one can download such snapshot files.

Reply:
When you say all models starting with @, do you mean drop any items in the 'model' column which start with @ or something different?

Reply:
yes, I meant drop all rows where the `model` entry starts with `@`